crowdsec-docs/docs/intro.mdx

```diff
@@ -7,17 +7,25 @@ id: intro
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import AcademyPromo from '@site/src/components/AcademyPromo';
 
+<!-- @tko
+
+ - replace the image with a gif 
+ - edit the image to have the same background color
+ - add the cool miniature for the console
+
+-->
+
 <div style={{display: 'flex'}}>
     <div style={{textAlign: 'center', flex: '1'}}>
-    <img width="800" height="auto" src={useBaseUrl('/img/crowdsec_ecosystem.png')} />
+    <img width="800" height="auto" src={useBaseUrl('/img/simplified_SE_overview.png')} />
     </div>
 </div>
 &nbsp;
 &nbsp;
 
-The [CrowdSec Security Engine](https://github.com/crowdsecurity/crowdsec) is an open-source, lightweight software that detects and blocks malicious actors from accessing your systems at various levels, using log analysis and threat patterns called scenarios. 
+The [CrowdSec Security Engine](https://github.com/crowdsecurity/crowdsec) is an open-source, lightweight software that detects and blocks malicious actors from accessing your systems at various levels, using log and HTTP Requests analysis with threat patterns called scenarios. 
 
-CrowdSec is a modular framework, offering a variety of [popular scenarios](https://app.crowdsec.net/hub/collections). Users can choose their protection scenarios and deploy [Remediation Components](https://app.crowdsec.net/hub/bouncers) to block malicious access.
+CrowdSec is a modular framework, offering a variety of [scenarios](https://app.crowdsec.net/hub/collections). Users can choose their protection scenarios and deploy [Remediation Components](https://app.crowdsec.net/hub/bouncers) to block malicious access.
```

Comment:
This sentence is not very clear.
- Are we using the word framework here ? it's a bit confusing
- We should define that they are behavior or detection scenarios and we could also mention appsec rules ehre
- - also we're talking about protection later and deploy remediation components (the ppl don't know about that yet, we should ease them into it by talking a higher level discourse here)

=== GPT assisted alternative:
CrowdSec is a modular solution designed to detect and respond to security threats. With a range of [behavior-based detection scenarios, including application security (AppSec) rules](https://app.crowdsec.net/hub/collections), users can choose what best fits their protection needs. Additionally, CrowdSec provides tools, known as [remediation components](https://app.crowdsec.net/hub/bouncers), to help block malicious access based on the chosen detection criteria.

== shorter ===
CrowdSec is a modular security tool offering [behavior-based detection, including AppSec rules](https://app.crowdsec.net/hub/collections), and optional components to block threats called [Remediation Components](https://app.crowdsec.net/hub/bouncers)